config/config_test.go

```diff
@@ -428,6 +428,46 @@ var _ = Describe("BlobstoreClient configuration", func() {
 			})
 		})
 	})
+
+	Describe("returning the alibaba cloud endpoint", func() {
```

Comment:
These tests do not seem related to the PR. Is there a reason that they should be added as a part of this change set? If not can you split them into a separate PR if you would like us to add them.

Reply:
Yes, you are right. These tests should be split them into a separate PR. Thanks a lot.